core/scrollbar.js

```diff
@@ -190,22 +189,36 @@ Blockly.Scrollbar = function(workspace, horizontal, opt_pair) {
   this.workspace_ = workspace;
   this.pair_ = opt_pair || false;
   this.horizontal_ = horizontal;
-  this.oldHostMetrics_ = {};
+  this.oldHostMetrics_ = null;
 
   this.createDom_();
 
+  this.scrollViewSize_ = 0;
+  this.handleLength_ = 0;
+  this.handlePosition_ = 0;
+  this.isVisible_ = true;
+
+  this.yCoordinate = 0;
```

Comment:
These two properties don't appear to be used external to this file, so they can be made private.
They also look like an easy pair to convert to a goog.math.Coordinate.